code/modules/jobs/job_types/legion.dm

```diff
@@ -622,10 +620,11 @@ commented out pending rework*/
 /datum/outfit/loadout/vexbear
 	name = "Mountain Bear"
 	head = /obj/item/clothing/head/helmet/f13/legion/vet/combvexil
+	suit_store = /obj/item/gun/ballistic/shotgun/automatic/combat/shotgunlever
 	backpack_contents = list(
 		/obj/item/melee/unarmed/tigerclaw = 1,
 		/obj/item/reagent_containers/pill/patch/healingpowder/berserker = 2,
-		/obj/item/book/granter/martial/berserker = 1
```

Comment:
I actually fixed berserker so it wouldn't be op (?) anymore. Now it deals less damage than the veteran fireaxe. It'll down 50/50 in five hits, no armor in 3 hits unless you get the 10% critical strike. So I guess it doesn't have to be removed anymore. #446

Reply:
> I actually fixed berserker so it wouldn't be op (?) anymore. Now it deals less damage than the veteran fireaxe. It'll down 50/50 in five hits, no armor in 3 hits unless you get the 10% critical strike. So I guess it doesn't have to be removed anymore. #446

This needed to happen. Good.

Reply:
👍🏻

Reply:
Alright, I can give it back.